src/AppBundle/Controller/PostController.php

```diff
@@ -30,4 +30,16 @@ public function onlPostViewAction($postId)
     {
 
     }
+
+    public function testJenkinsCI()
```

Comment:
**Reporter**: CHECKSTYLE
**Rule**: PEAR.Commenting.FunctionComment.Missing
**Severity**: ERROR
**File**: /var/lib/jenkins/workspace/test2/src/AppBundle/Controller/PostController.php L34

Missing doc comment for function testJenkinsCI()


*<this is a auto generated comment from violation-comments-lib F7F8ASD8123FSDF>* *<a-1651865324>*